src/app/client/src/app/modules/learn/components/batch/batch-details/batch-details.component.html

```diff
@@ -1,5 +1,5 @@
 <!-- Join Course Table -->
-<sui-modal *ngIf="showJoinModal && !courseMentor" [mustScroll]="true" [isClosable]="true" [transitionDuration]="0"
+<sui-modal *ngIf="showJoinModal" [mustScroll]="true" [isClosable]="true" [transitionDuration]="0"
```

Comment:
This will allow, COURSE_MENTOR also to join for the batch right? Is it mentioned in the story?